backstop/tests/header.html

```diff
@@ -0,0 +1,969 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html id="theme">
+  <head>
+    <title>Header | iTwinUI</title>
+    <link
+      rel="icon"
+      href="../assets/favicon.ico"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/global.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/button.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/inputs.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/user-icon.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/header.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <script src="../assets/theme.js"></script>
+    <style>
+      .iui-thumbnail {
+        background-image: url('../assets/image-test.jpg');
+      }
+
+      .demo-side-navigation {
+        background-color: var(--iui-color-background-1);
+        width: 72px;
+        height: 55px;
+        border-right: 1px solid var(--iui-color-background-5);
+        border-bottom: 1px solid var(--iui-color-background-5);
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+
+      .demo-side-navigation-icon {
+        width: 24px;
+        height: 24px;
+        fill: var(--iui-icons-color-actionable);
+      }
+
+      .iui-input {
+        border-radius: 22px;
+        width: 20vw;
+        background-color: var(--iui-color-background-2);
+      }
+
+      body {
+        background-color: var(--iui-color-background-2) !important;
```

Comment:
Why this one is needed?

Reply:
The screenshot below is from a UX Tuesday presentation.  The header and side navigation will typically be placed above background-2 where as most other components can be placed above background-1 _or_ background-2.
![Screen Shot 2021-05-04 at 11 16 42 AM](https://user-images.githubusercontent.com/849817/117026957-110a2b00-acb1-11eb-97aa-6748aa6bdd50.png)

Reply:
So the user will have to put this background color on their own?

Maybe we need to make background-2 the default for `iui-body`